### ISSUE TITLE
A beta version 0.2.3 published

### ISSUE THREAD

**Issue by StephenChou1017** [StephenChou1017/react-big-scheduler#52]
Hi all,

Just now I published one beta version 0.2.3, to test the using of react-big-scheduler. Generally it's almost the same with the latest version 0.2.2, the only difference is that 0.2.3 tries to fix leaking antd global CSS rules described in [here](https://github.com/StephenChou1017/react-big-scheduler/pull/19). I hope you can try this beta version, mainly in the two cases:

1. All web site pages are antd style compatible and use antd global CSS rules;
2. The web site pages have their own global CSS rules, only a few pages use the Scheduler component.

Feel free to tell me if any issues. Thanks.

**Comment by Saikat-Sinha** [comment]
@StephenChou1017 can you please publish 0.2.3 to npm?

**Comment by StephenChou1017** [comment]
@vienpndn 
Yes, I found this problem too, I published it in beta mode instead of latest mode. I'll publish a latest version.

**Comment by vienpndn** [comment]
@StephenChou1017 Thanks, sound good to me. I hope you can do it soon.

**Comment by StephenChou1017** [comment]
@Saikat-Sinha @vienpndn 
Hi, I've just published 0.2.4:-)

**Comment by vienpndn** [comment]
Great !!!, thanks so much @StephenChou1017  :)

**Comment by Saikat-Sinha** [comment]
@StephenChou1017 Thanks buddy 🥇
